### ISSUE TITLE
Training on custom dataset 

### ISSUE THREAD

**Issue by eyebies** [xuannianz/DifferentiableBinarization#6]
Thanks for sharing the code. I am trying to finetune for custom data and was able to get the loss to under 2.0 . however, the inference does not work. Am I missing anything? Here are a few sample predictions! training is converging!
![3d9a47a2-5156-11ea-84de-02423f4d0594](https://user-images.githubusercontent.com/3131247/74633029-6c620300-5169-11ea-8b02-2a27a3326f1b.jpg)
![3d9a47a2-5156-11ea-84de-02423f4d0594](https://user-images.githubusercontent.com/3131247/74633123-9e736500-5169-11ea-9314-0bd34dc31bd8.jpg)
![3d9a47ab-5156-11ea-84de-02423f4d0594](https://user-images.githubusercontent.com/3131247/74633129-a501dc80-5169-11ea-96d7-b1a3c639966c.jpg)
![3d9a47ab-5156-11ea-84de-02423f4d0594](https://user-images.githubusercontent.com/3131247/74633138-aa5f2700-5169-11ea-8b06-4e70ff74f21e.jpg)

**Comment by its-jd** [comment]
Hello, how did you convert the groud truth files for custom dataset? is there any implementation for converting the ground truth availabel?
